src/components/navigation/Pagination.tsx

```diff
@@ -0,0 +1,148 @@
+import classnames from 'classnames';
```

Comment:
This is unchanged from `PaginationNavigation` in the client aside from the rename and a few minor JSDoc additions.